setup.py

```diff
@@ -1,3 +1,10 @@
 from setuptools import setup
 
-setup()
+
+def local_scheme(version):
+    return ""
+
+
+setup(
+    use_scm_version={"local_scheme": local_scheme},
```

Comment:
This seems like a good change. It would be better in `pyproject.toml` under the `tool.setuptools_scm` section, since that's where the other setup configuration items live.